src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -327,6 +327,71 @@ export default defineConfig({
 });
 ```
 
+### Local development features
+
+When running `astro dev`, the adapter enables several Netlify platform features to ensure the environment matches production as closely as possible. These include:
+
+- A local [Netlify Image CDN](https://docs.netlify.com/build/image-cdn/overview/) server. This is used for [images](#netlify-image-cdn-support) by default.
+- A local [Netlify Blobs](https://docs.netlify.com/build/data-and-storage/netlify-blobs/) server. This is used for [sessions](#sessions) by default
+- [Redirects, rewrites](https://docs.netlify.com/manage/routing/redirects/overview/) and [headers](https://docs.netlify.com/manage/routing/headers/) from your Netlify config
+- Access to [Netlify Edge Context](/en/guides/integrations-guide/netlify/#accessing-edge-context-from-your-site) in on-demand pages
+- [Environment variables](https://docs.netlify.com/build/environment-variables/overview/) from your Netlify site
+
+These work best when your local site is [linked to a Netlify site](https://docs.netlify.com/api-and-cli-guides/cli-guides/get-started-with-cli/#link-and-unlink-sites) using `netlify link`.
+
+You can enable or disable some of these features using the [`devFeatures`](#devfeatures) option in your adapter configuration. By default, all features are enabled except for environment variables.
+
+#### `devFeatures`
+
+<p>
+**Type:** `boolean | object`<br />
+**Default:** `{ images: true, environmentVariables: false }`<br />
+<Since v="6.5.1" pkg="@astrojs/netlify"/>
+</p>
+
+The `devFeatures` option can be either a boolean to enable or disable all features, or an object to enable specific features.
+
+```js title="astro.config.mjs" ins={7-12}
+import { defineConfig } from 'astro/config';
+import netlify from '@astrojs/netlify';
+
+export default defineConfig({
+  // ...
+  adapter: netlify({
+    devFeatures: {
+      // Enable Netlify Image CDN support in dev. Defaults to true.
+      images: true, 
```

Comment:
```suggestion
      images: false, 
```
We typically wouldn't show setting to true, since that's the default and no one would need to do this.

But this brings up a good point: if they *do* configure an object, *must* they specify values for both (because this overrides), or do the defaults work when not all properties are specified?